lib/msal-angular/src/msal.broadcast.service.spec.ts

```diff
@@ -59,6 +66,238 @@ describe('MsalBroadcastService', () => {
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }, 
+      {
+        eventType: EventType.LOGIN_START,
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }
+    ];
+    let firstIndex = 0;
+
+    subscription = broadcastService.msalSubject$.subscribe((result) => {
+      expect(result.eventType).toEqual(expectedMsalSubjectFirstSubscription[firstIndex].eventType);
+      expect(result.interactionType).toEqual(expectedMsalSubjectFirstSubscription[firstIndex].interactionType);
+      expect(result.payload).toEqual(expectedMsalSubjectFirstSubscription[firstIndex].payload);
+      expect(result.error).toEqual(expectedMsalSubjectFirstSubscription[firstIndex].error);
+      expect(result.timestamp).toBeInstanceOf(Number);
+      if (firstIndex === (expectedMsalSubjectFirstSubscription.length - 1)) {
+        return;
+      } else {
+        firstIndex++;
+      }
+    });
+
+    // @ts-ignore
+    msalInstance.eventHandler.emitEvent(EventType.INITIALIZE_START, InteractionType.Redirect);
+    // @ts-ignore
+    msalInstance.eventHandler.emitEvent(EventType.LOGIN_START, InteractionType.Redirect);
+
+    subscription.unsubscribe();
+
+    let index = 0;
+    const expectedMsalSubject = [
+      {
+        eventType: EventType.LOGIN_START,
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }, 
+      {
+        eventType: EventType.HANDLE_REDIRECT_START,
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }, 
+      {
+        eventType: EventType.HANDLE_REDIRECT_END,
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }
+    ];
+
+    const newSubscription = broadcastService.msalSubject$.subscribe((result) => {
+      expect(result.eventType).toEqual(expectedMsalSubject[index].eventType);
+      expect(result.interactionType).toEqual(expectedMsalSubject[index].interactionType);
+      expect(result.payload).toEqual(expectedMsalSubject[index].payload);
+      expect(result.error).toEqual(expectedMsalSubject[index].error);
+      expect(result.timestamp).toBeInstanceOf(Number);
+      if (index === (expectedMsalSubject.length - 1)) {
+        done();
+      } else {
+        index++;
+      }
+    });
+
+    // @ts-ignore
+    msalInstance.eventHandler.emitEvent(EventType.HANDLE_REDIRECT_START, InteractionType.Redirect);
+    // @ts-ignore
+    msalInstance.eventHandler.emitEvent(EventType.HANDLE_REDIRECT_END, InteractionType.Redirect);
+
+    newSubscription.unsubscribe();
+  });
+
+  it('does not broadcasts previous events if MsalBroadcastConfig is not set', (done) => {
+    const expectedMsalSubjectFirstSubscription = [
+      {
+        eventType: EventType.LOGIN_START,
+        interactionType: InteractionType.Redirect,
+        payload: null,
+        error: null,
+      }, 
```

Comment:
super nitpick: I've noticed that you have inconsistently used trailing commas all throughout this review.

Reply:
Thanks for the review and for pointing this out! Fixed the trailing commas.